Utils/KeyboardHelper.swift

```diff
@@ -41,6 +41,7 @@ public struct KeyboardState {
 
 public protocol KeyboardHelperDelegate: class {
```

Comment:
If you want, you can have optional methods for the protocol by adding @objc in front of public so you don't need the empty delegate methods.